src/main/java/seedu/address/ui/PersonListPanel.java

```diff
@@ -46,4 +46,11 @@ protected void updateItem(Person person, boolean empty) {
         }
     }
 
+    /**
+     * Manually refreshes the ListView to update Favourite status consistently
+     */
+    public void refresh() {
+        personListView.refresh();
+    }
```

Comment:
Perhaps you might want to remove this as well since it is not in use already?

Reply:
Alright thanks, will push the changes now